underfs/pom.xml

```diff
@@ -39,7 +39,7 @@
       <dependency>
         <groupId>net.java.dev.jets3t</groupId>
         <artifactId>jets3t</artifactId>
-        <version>0.8.1</version>
+        <version>0.9.4</version>
```

Comment:
Yes, I made a little modify to s3a module, to accept user define entry point like “s3.cn-north-1.amazonaws.com.cn” and it works perfect for signature v4 request authentication. Should I make another pr only include those s3a project changes?